.github/actions/setup-nox/action.yaml

```diff
@@ -4,23 +4,23 @@ description: Prepare and cache nox
 runs:
   using: composite
   steps:
-    - uses: actions/setup-python@v3
+    - uses: actions/setup-python@v4
```

Comment:
I don't really how Github actions / dependabot work, so I can't offer an opinion about why dependabot didn't notice...

Reply:
Hmmm, it's been out for a while, let me have a look at why it's not updating it. Thanks for finding this!

Reply:
It requires an explicit config in the `.dependabot.yml` config: https://github.com/dependabot/dependabot-core/issues/4178#issuecomment-1118492006

If you want to add that, feel free, otherwise I'm happy to do it

Reply:
You should do it. I'm at the very edge of my understanding of how all this stuff interlocks... Thanks.